dot-studio/golang

```diff
@@ -260,8 +299,8 @@ function go_debug_mode() {
     return 1
   fi
 
-  # Install the debugging tooli
-  install_go_tool github.com/derekparker/delve/cmd/dlv
+  # Install the debugging tooling
+  GOFLAGS="-mod=readonly" install_go_tool github.com/go-delve/delve/cmd/dlv
```

Comment:
We use `-mod=readonly` because `dlv` is not a hard dependency. We use it only for development so,
it is indeed read-only and this flag will avoid making the dependency persistent.